res/values/strings.xml

```diff
@@ -837,7 +837,7 @@
     <!-- android specific strings, developers: please take care to remove strings that are no longer used! -->
     <string name="pref_background_notifications">Background notifications</string>
     <string name="pref_background_notifications_explain">Uses a background connection to your server and requires ignored battery optimizations.</string>
-    <string name="pref_background_notifications_rationale">To maintain connection to your email server and receive messages in the background, ignore battery optimizations in the next step.\n\nDelta Chat uses few resources and takes care not to drain your battery.</string>
+    <string name="pref_background_notifications_rationale">To maintain connection to your e-mail server and receive messages in the background, ignore battery optimizations in the next step.\n\nDelta Chat uses few resources and takes care not to drain your battery.</string>
```

Comment:
@r10s It is never an "e-mail server", always "mail server".
"The server that holds your e-mails" avoids this problem.

Reply:
interesting, indeed, there are few results for "e-mail server" - but still some, eg. https://www.ionos.com/office-solutions/email-server

so, i am wondering if people really will take care, esp. as it looks more consistent the way we're doing it currently, but at the end, i would not really care.

("The server that holds your e-mails" seems to be too long here, things are already complicated enough in this sentence)

Reply:
I would argue use by people opting for "About IONOS Mail Server" just below it solidifies my point.
What I would do is
"Turn off battery optimizations to receive messages as they arrive." Y/N
Fluff is bad UI, and harder yet to translate than it is to read. Delta Chat seems to target users wanting technicalities semi-explained at length, when there are no such users.